### PR TITLE
Added Wekan Source Code

### PR DESCRIPTION
**Pull request by bnhn**
Added Wekan Source Code

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Please resolve the merge conflicts @bnhn

File: README.md [hunk 1]
```diff
@@ -105,3 +105,4 @@ Even if you are just a developer, manager or co-founder looking for a sample app
 |[CoderMania](https://github.com/Elfoslav/codermania) | An E-Learning Platform | [http://www.codermania.com/](http://www.codermania.com/)|
 | [Telescope](https://github.com/TelescopeJS/Telescope) | Lauch Sites Easily | [http://www.telescopeapp.org/](http://www.telescopeapp.org/)|
 | [Loop Pulse](https://github.com/clionelabs/looppulse.web.manage)| Data Analysis Platform| [http://www.looppulse.com/](http://www.looppulse.com/)|
+| [Wekan](https://github.com/wekan/wekan) | A Trello-like collaboration platform | [https://wekan.io](https://wekan.io) |
```
Comment (bnhn) [inline README.md]: Added Wekan Source Code

**Comment by bnhn** [comment]
I think I have done it now, please check and lemme know. Thanks

**Comment by bnhn** [comment]
The conflicts say "only those with write access can merge PRs", how do I
resolve this please

On Mon, May 9, 2016 at 3:53 PM Prosper Otemuyiwa <notifications@github.com>
wrote:

> Please resolve the merge conflicts @bnhn <https://github.com/bnhn>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/unicodeveloper/awesome-opensource-webapps/pull/11#issuecomment-217905565>
>
-- 

Regards,
Rayy Benhin

**Comment by unicodeveloper** [comment]
1. Check if there are any conflicts in the file, 
2. If there are, remove them, commit.
3. Then do a git rebase like so `git pull --rebase origin master"

**Pull request closed without merging**